src/20241030_AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain/AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain_20241030.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3BNB, AaveV3BNBAssets} from 'aave-address-book/AaveV3BNB.sol';
+import {AaveV3PayloadBNB} from 'aave-helpers/src/v3-config-engine/AaveV3PayloadBNB.sol';
+import {EngineFlags} from 'aave-v3-origin/contracts/extensions/v3-config-engine/EngineFlags.sol';
+import {IAaveV3ConfigEngine} from 'aave-v3-origin/contracts/extensions/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+/**
+ * @title Onboard wstETH to Aave V3 on BNB Chain
+ * @author ACI
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x752c396a86f1f9b60d3e43b7ed223d9d2f66014e03b6b3eb7ac59e8a169d1fe5
+ * - Discussion: https://governance.aave.com/t/arfc-onboard-wsteth-to-aave-v3-on-bnb-chain/18501
+ */
+contract AaveV3BNB_OnboardWstETHToAaveV3OnBNBChain_20241030 is AaveV3PayloadBNB {
+  using SafeERC20 for IERC20;
+
+  address public constant wstETH = 0x26c5e01524d2E6280A48F2c50fF6De7e52E9611C;
+  uint256 public constant wstETH_SEED_AMOUNT = 4e16;
+
+  function _postExecute() internal override {
+    IERC20(wstETH).forceApprove(address(AaveV3BNB.POOL), wstETH_SEED_AMOUNT);
```

Comment:
on the other pr https://github.com/bgd-labs/aave-proposals-v3/pull/501/files#diff-8b9426a3cb9f826a6585517b4e18f50b4b0302f890ae08bff85789e2e2d22568R29 you where also adding the emission admin. should this still be added? if so, it should also be added on the md

Reply:
Ah indeed, I missed this when using the generator, I'll fix this, thanks!

Reply:
Now the only diference I see on the _postExecute() of the old PR is this:
`AaveV3BNB.COLLECTOR.transfer(wstETH, address(this), wstETH_SEED_AMOUNT);`
As the [BNB Executor](https://bscscan.com/address/0x9390B1735def18560c509E2d0bc090E9d6BA257a) has already the 4e16 wstETH needed, I suppose I don't need to add this line.

Reply:
yup no need